.pipelines/pipeline.user.windows.yml

```diff
@@ -118,6 +118,10 @@ build:
             - 'modules\launcher\Plugins\Microsoft.Plugin.Program\Wox.Infrastructure.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.Program\Wox.Plugin.dll'
             - 'modules\launcher\Plugins\Microsoft.Plugin.Program\Telemetry.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Registry\Microsoft.PowerToys.Run.Plugin.Registry.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Registry\Wox.Infrastructure.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Registry\Wox.Plugin.dll'
+            - 'modules\launcher\Plugins\Microsoft.PowerToys.Run.Plugin.Registry\Telemetry.dll'
```

Comment:
This is the telemetry dll for C++ projects, the other plugins use `ManagedTelemetry.dll`